system_test/m1_facilitator/WithdrawSystemTest.ts

```diff
@@ -154,21 +148,20 @@ export default class WithdrawSystemTest {
       Logger.info('Final balances captured');
 
       const erc20Gateway = Mosaic.interacts.getERC20Gateway(originWeb3, config.chains.origin.gateway);
-      // wait for facilitator to finish the job
 
       await utils.waitForCondition(
         async (): Promise<boolean> => {
           for (let j = 0; j < messageHashes.length; j += 1) {
             const isDeclared = await erc20Gateway.methods.inbox(messageHashes[j]).call();
+            Logger.debug(`Message status on origin chain is ${isDeclared} for message hash ${messageHashes[j]}`);
             if (!isDeclared) {
               return false;
             }
           }
           return true;
         },
-        1000, // todo: fix me
-        // 5 * 60 * 1000,
-        6,
+        1000 * 60, // poll in 60 sec
+        5, //  max retry 5000
```

Comment:
Take these values from the constant variables.

Reply:
Done